npeps/npep-137-conformance-profiles.md

```diff
@@ -37,10 +37,19 @@ the results.
 
 ## Introduction
 
-Currently, the conformance tests are grouped into `CoreFeatures` and
-`ExtendedFeatures`. The support for features that fall under the `CoreFeatures`
+Currently, the conformance tests are grouped into `StandardFeatures` and
+`ExperimentalFeatures`.
+
+* A feature is considered to be `Standard` and is part of the `StandardCRD` if it is
+stable and has had more than 6 months of soak time and has two or more implementations
+implementing it.
+* A feature is considered to be `Experimental` and is part of the `ExperimentalCRD` if
+it is relatively new, not stable enough as decided by the community or doesn't have at
+least two implementations implementing it.
+
+The support for features that fall under the `StandardFeatures`
 are a requirement for conformant implementations, while the support for features that fall
-under `ExtendedFeatures` are optional and do not gate API conformance for implementations.
+under `ExperimentalFeatures` are optional and do not gate API conformance for implementations.
```

Comment:
But if an implementation is going to implement an Experimental feature, then it needs to implement it in a conformant way, right? Otherwise we aren't getting proper feedback from that implementation.

Maybe this doesn't belong in _this_ doc, but we need to explain conformance better somewhere...

Reply:
hmm in some sense I think that is what we do today. even for experimental features, the conformance tests must pass which is what gives out this conformance report. So the wording needs to be updated here.

Reply:
took a re-stab at it.. let's see if that's a bit better?